README.md

```diff
@@ -89,20 +88,26 @@ protocol should provide additional guarantees in the event that the tolerance
 capacity is exceeded and the consensus fails.  This is especially necessary in
 public economic systems, where Byzantine behaviour can have substantial
 financial reward.  The most important such guarantee is a form of
-_accountability_, where the processes that caused the consensus to fail can
+_fork-accountability_, where the processes that caused the consensus to fail (ie. 
+caused clients of the protocol to accept different values - a fork) can
 be identified and punished according to the rules of the protocol, or, possibly,
 the legal system.  When the legal system is unreliable, validators can be forced
 to make security deposits in order to participate, and those deposits can be
 revoked, or slashed, when malicious behaviour is detected [\[10\]][10].
 
-Tendermint is a Byzantine fault-tolerant (BFT) consensus protocol for
-asynchronous networks, notable for its simplicity, performance, and
-fork-accountability.  The protocol requires a fixed, known set of N validators,
-where the ith validator is identified by its public key, `V_i`. Validators
-attempt to come to consensus on one block at a time, where a block is a list of
-transactions.  Consensus on a block proceeds in rounds. Each round has a
+Note this is much unlike Bitcoin, where forking is a regular occurence due to network asynchrony
+and the probabilistic nature of finding partial hash collissions. 
+Since in many cases, a malicious fork is indistinguishable from a fork due to asynchrony,
+Bitcoin can not reliably implement fork-accountability.
```

Comment:
they kind of do. but forking is still a regular/expected occurrence, and its not punished explicitly ...